doc/getstarted/README.md

```diff
@@ -0,0 +1,110 @@
+# Get started 
```

Comment:
Consider adding a line that says something like "Mongoose supports CRUD notation, hence the name of the operations". We have that information somewhere, so we can also leave a link to it. It seems reasonable to have it stated again in the "newcomers" section.

Reply:
Load operations defenition? https://github.com/emc-mongoose/mongoose-base/tree/BASE-1389-documentation/doc/design/architecture#1-basic-terms

Reply:
It says nothing about CRUD. so the connection between "write" and "create" isn't obvious